app/dashboard/templates/profile_details.html

```diff
@@ -1,224 +1,182 @@
+                                        </div>
+                                        <div class="col-2">
+                                            <strong>{% trans "Status" %}</strong>
+                                        </div>
+                                        <div class="col-10 col-sm-8 col-md-4">
+                                            <strong>{% trans "Details" %}</strong>
+                                        </div>
+                                        <div class="col-4 col-md-4  d-none d-md-block">
+                                            <strong>{% trans "Amount" %}</strong>
+                                        </div>
+                                    </div>
+                                    {% for b in bounties %}
+                                    <div class="row bounty-row" data-ref-url="{{b.url}}">
+                                        <div class="col-2 d-none d-sm-block">
+                                            {% if profile.handle == b.fulfiller_github_username %} {% trans "Worker" %} {% elif profile.handle == b.bounty_owner_github_username%} {% trans "Funder" %} {% elif profile.handle == b.org_name %} {% trans "Repo
+                                            Owner" %} {% else %} {% trans "Worker" %} {% endif %}
+                                        </div>
+                                        <div class="col-2">
+                                            {{b.status}}
+                                        </div>
+                                        <div class="col-10 col-sm-8 col-md-4">
+                                            <img style="max-width: 30px;" src="{{b.local_avatar_url}}">
+                                            <h6>
+                                                {{b.title_or_desc | truncatechars:50}}
+                                            </h6>
+                                            <p class="bounty-desc">
+                                                {{b.desc}}
+                                            </p>
+                                        </div>
+                                        <div class="col-4 col-md-4  d-none d-md-block bounty-value">
+                                            <a class="btn btn-sm btn-darkBlue font-smaller-2">
+                                                {{b.value_true}} {{b.token_name}}, {{b.value_in_usdt}} USD
+                                            </a>
+                                        </div>
+                                    </div>
+                                    {% endfor %} {% endif %}
+                                </div>
                             </div>
                             <div>
-                            <h4>Repos</h4>
+                                <h4>Repos</h4>
                                 <ul class="repos">
                                     {% for r in profile.repos_data %}
-                                        <li>
```

Comment:
Don't the other files follow this ? It's easier to read when we conditional statements